config/deploy.rb

```diff
@@ -39,7 +39,7 @@
 # Default value for keep_releases is 5
 # set :keep_releases, 5
 
-desc "Write the current version to public/version.txt"
+desc "Write SHA of current deploy to /version.txt "
```

Comment:
There's an extra space before the closing quotation
```suggestion
desc "Write SHA of current deploy to /version.txt"
```